site/docs/operations/docker.md

```diff
@@ -186,6 +222,51 @@ using `-n jet-1` option (shortcut for `--cluster-name`):
 docker run -it -v "$(pwd)"/examples:/examples hazelcast/hazelcast-jet jet -t 172.17.0.2 -n jet-1 submit /examples/hello-world.jar
 ```
 
+You can view the contents of the default configuration file inside the
+container by running the following command:
+
+```bash
+docker run hazelcast/hazelcast-jet:{jet-version} cat /opt/hazelcast-jet/config/hazelcast.yaml
```

Comment:
Should we use `{jet-version}` here? Other docker commands are without version (i.e. they use latest)

Reply:
Right.